web/packages/teleport/src/Roles/RoleEditor/StandardEditor.tsx

```diff
@@ -636,6 +654,97 @@ export function AppAccessSpecSection({
   );
 }
 
+export function DatabaseAccessSpecSection({
+  value,
+  isProcessing,
+  onChange,
+}: SectionProps<DatabaseAccessSpec>) {
+  return (
+    <>
+      <Box mb={3}>
+        <Text typography="body3" mb={1}>
+          Labels
+        </Text>
+        <LabelsInput
+          disableBtns={isProcessing}
+          labels={value.labels}
+          setLabels={labels => onChange?.({ ...value, labels })}
+        />
+      </Box>
+      <FieldSelectCreatable
+        isMulti
+        label="Names"
+        toolTipContent="List of database names that this role is allowed to connect to"
+        isDisabled={isProcessing}
+        formatCreateLabel={label => `Name: ${label}`}
+        components={{
+          DropdownIndicator: null,
+        }}
+        value={value.names}
+        onChange={names => onChange?.({ ...value, names })}
+      />
+      <FieldSelectCreatable
+        isMulti
+        label="Roles"
+        toolTipContent="List of database roles for automatic user creation"
```

Comment:
I'm unsure what this means. Could you explain the automatic user creation to me?